src/_locales/en/messages.json

```diff
@@ -118,5 +118,17 @@
   "alert_new_event_added": {
     "message": "This event has just been added to your calendar: ",
     "description": "When a new event is added into the user's calendar, an alert message will be displayed for the user to click on it and see the details"
+  },
+  "option_notification_show": {
+    "message": "Show notifications.",
+    "description": "Show notifications before an event starts."
+  },
+  "your_event_starts_in": {
```

Comment:
Since many languages may not have the “minutes” part after the rest of the message, the better way to structure such messages is to use placeholders. E.g.

      "message": "$eventName$ starts in $minutes$ minutes",
        "placeholders": {
          "eventName": {
            "content": "$1",
            "example": "SomeEvent"
          },
          "minutes": {
            "content": "$1",
            "example": "42"
          }
        }